hw06-ATM/src/main/java/homework/Controller.java

```diff
@@ -0,0 +1,60 @@
+package homework;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+public class Controller {
```

Comment:
```suggestion
public class AtmController {
```

Название `Controller` очень общее, не понятно, что за контроллер.